src/Omnipay/Common/Message/AbstractResponse.php

```diff
@@ -182,8 +183,9 @@ public function getRedirectResponse()
             throw new RuntimeException('This response does not support redirection.');
         }
 
+        /** @var $this RedirectResponseInterface */
         if ('GET' === $this->getRedirectMethod()) {
-            return HttpRedirectResponse::create($this->getRedirectUrl());
+            return new RedirectResponse($this->getRedirectUrl());
         } elseif ('POST' === $this->getRedirectMethod()) {
```

Comment:
This doesn't need to be `elseif`, since we're returning in the above if statement